addons/spreadsheet/static/tests/utils/global_filter.js

```diff
@@ -0,0 +1,11 @@
+/** @odoo-module */
+/** @type FilterPayload */
```

Comment:
this is useless if the type is never defined (with `@typedef`)
I think you can remove it